src/layer/place.js

```diff
@@ -208,6 +208,175 @@ export const city = {
   metadata: {},
 };
 
+export const suburb = {
+  id: "place_suburb",
+  type: "symbol",
+  paint: {
+    "text-color": Color.waterLabel,
+    "text-halo-color": labelHaloColor,
+    "text-halo-width": [
+      "interpolate",
+      ["exponential", 1.2],
+      ["zoom"],
+      3,
+      1.5,
+      6,
+      2.5,
+    ],
+    "text-halo-blur": labelHaloBlur,
+  },
+  filter: ["==", ["get", "class"], "suburb"],
+  layout: {
+    "text-font": ["Americana-Regular"],
+    "text-size": {
+      base: 1.2,
+      stops: [
+        [10, 12],
+        [12, 15],
+        [14, 18],
+      ],
+    },
+    "text-field": Label.localizedName,
+    "text-padding": 1,
+    "text-transform": "uppercase",
+    "text-letter-spacing": {
+      base: 0.04,
+      stops: [
+        [11, 0.04],
+        [12, 0.08],
+        [13, 0.2],
+        [14, 0.4],
+      ],
+    },
+    "text-variable-anchor": ["center"],
+    "text-radial-offset": [
+      "interpolate",
+      ["exponential", 1.6],
+      ["zoom"],
+      3,
+      0.5,
+      7,
+      3,
+    ],
+    "text-max-width": 6,
+  },
+  source: "openmaptiles",
+  maxzoom: 15,
+  minzoom: 11,
+  "source-layer": "place",
+};
+
+export const quarter = {
+  id: "place_quarter",
+  type: "symbol",
+  paint: {
+    "text-color": Color.waterLabel,
```

Comment:
Even though the same color is used, can we create a new variable for the color of these place labels?  It would be surprising for a future water label color change to also change the color of neighborhood and suburb labels as a side effect.  Maybe `suburbQuarterNeighborhoodLabel` or `citySubAreaLabel`.  Or if we may use this color for other area labels too (like counties and states) then just `placeAreaLabel` could make sense.

Reply:
I've defined a new urbanSubAreaLabel color variable, same color as waterLabel